MVP/sample_job.yaml

```diff
@@ -0,0 +1,29 @@
+- job-template:
+    name: '{name}-{component}'
+    defaults: defaults-build
+    #node: 'jslave-of-your-own'
+    properties:
+        - ownership:
+            owner: lilu
+            co-owners:
+                - lilu
+                - lilu
+    triggers:
+        - ci-trigger:
+            jms-selector: "name = '{component}' AND CI_TYPE = 'brew-tag' AND (tag LIKE 'RHEL-6._-candidate' OR tag LIKE 'rhel-7._-candidate')"
+    builders:
+        - shell: |
+            set -x
+
+            # Here goes your shell commands for submitting beaker job here, and save the job ID into env var BKR_JOBID
+            # E.g. `bkr workflow-tomorrow --distro=$distro --task-file=$taskfile $other_args`
+
+            BKR_JOBID="J:1000000"
```

Comment:
It's for demo purpose.
We'll remove this as discussed above (adding an example about parsing job id into BKR_JOBID).